examples/nnp_training_ignite.py

```diff
@@ -59,10 +60,9 @@
 # Instead of manually specifying hyperparameters as in :ref:`training-example`,
 # here we will load them from files.
 const_file = os.path.join(path, '../torchani/resources/ani-1x_8x/rHCNO-5.2R_16-3.5A_a4-8.params')  # noqa: E501
-sae_file = os.path.join(path, '../torchani/resources/ani-1x_8x/sae_linfit.dat')  # noqa: E501
 consts = torchani.neurochem.Constants(const_file)
 aev_computer = torchani.AEVComputer(**consts)
-energy_shifter = torchani.neurochem.load_sae(sae_file)
+energy_shifter = torchani.neurochem.load_sae(None)
```

Comment:
What does `load_sae(None)` do?

Reply:
oops! that was a mistake! fixed in https://github.com/aiqm/torchani/pull/283/commits/f6346846ad90f003e1c789d16586327b2768a643